cmd/proxy/main.go

```diff
@@ -28,7 +28,7 @@ var (
 	httpProxyAddr              = flag.String("http_addr", "localhost:3128", "address for HTTP proxy")
 	tlsProxyAddr               = flag.String("tls_addr", "localhost:3129", "address for TLS proxy")
 	ctrlAddr                   = flag.String("ctrl_addr", "localhost:3127", "address for administrative endpoint")
-	dockerAddr                 = flag.String("docker_addr", "", "address for docker proxy endpoint")
+	dockerAddr                 = flag.String("docker_addr", "", "address for docker proxy endpoint in the format host:port for tcp, or unix:///file for unix domain sockets.")
```

Comment:
we might also support tcp://... since I believe DOCKER_HOST does the same: https://docs.docker.com/reference/cli/docker/#host